test/legacy/fairroot.sh

```diff
@@ -10,7 +10,8 @@ git clone -b ${version} https://github.com/FairRootGroup/FairRoot $sourcedir
 pushd $sourcedir
 export FAIRROOTPATH="$(realpath ./install)"
 cmake -S. -Bbuild \
-  -DCMAKE_INSTALL_PREFIX=$FAIRROOTPATH
+  -DCMAKE_INSTALL_PREFIX=$FAIRROOTPATH \
+  -DCMAKE_CXX_STANDARD=17
```

Comment:
Can you please inspect the compile commands on FairRoot to see, which `-std=` is actually getting in there?

Reply:
I should remove this commit again, it is redundant. FairRoot will take the -std flag exported by the `fairsoft-config` script.